Source/Core/Core/HW/WiimoteReal/IOWin.cpp

```diff
@@ -789,6 +790,10 @@ int WiimoteWindows::IOWrite(const u8* buf, size_t len)
 {
 	return _IOWrite(m_dev_handle, m_hid_overlap_write, m_stack, buf, len, nullptr);
 }
+char* WiimoteWindows::Address()
+{
+        return m_devicepath.c_str();
```

Comment:
That returns a const char*. You lose the const safety here, I'm surprised it even compiles.